src/index.js

```diff
@@ -6,3 +6,6 @@ export { useElementShouldClose } from './hooks/use-element-should-close';
 
 // Utilities
 export { normalizeKeyName } from './browser-compatibility-utils';
+
+// Test utilities
+export { checkAccessibility } from './test-util/accessibility';
```

Comment:
We'll need to put test utilities in a separate module than runtime utilities, otherwise Browserify is going to try and include all the transitive dependencies of the test utilities, such as enzyme and axe-core into the app bundle!

If we had a smarter bundler which supports tree shaking (eg. modern versions of Webpack, Rollup or Parcel) then they might be able to avoid doing this when the test utilities are not directly used. However even if that worked I'd still advise having a separate entry point for test dependencies.

To specify dependencies properly, the test utilities also ought to declare their Enzyme and axe-core peer dependencies. We don't really want the frontend-shared package to depend on those, so it is likely that test utilities should really go in their own package, which this repository then imports. The `frontend-toolkit/` repo might be an appropriate place for this, since it already has other development-only packages.